odata-protocol/11.5 Operations.md

```diff
@@ -505,11 +505,13 @@ and MAY include a [`Retry-After`](#HeaderRetryAfter) header to be used for a sub
 `Location` header and optional `Retry-After` header may or may not
 contain the same values as returned by the previous request.
 
-A `GET` request to the status monitor resource returns [`200 OK`](#ResponseCode200OK) once the
-asynchronous processing has completed. For OData 4.01 or greater
-responses, or OData 4.0 requests that include an `Accept` header that
+Once the asynchronous processing has completed, a `GET` request to the status monitor resource returns [`200 OK`](#ResponseCode200OK),
+or [`204 No Content`](#ResponseCode204NoContent) if the request included a `Prefer` header with a value of `return=minimal` that was applied by the service.
+For OData 4.01 or greater responses, or OData 4.0 requests that include an `Accept` header that
 does not specify `application/http`, the response MUST include the
-[`AsyncResult`](#HeaderAsyncResult) response header. Any other headers,
+[`AsyncResult`](#HeaderAsyncResult) response header.
+A `204 No Content` response MUST include a `Location` header that contains the status monitor resource URL, which may or may not be the current request's URL.
```

Comment:
```suggestion
A `204 No Content` response MUST include a `Location` header that contains the status monitor resource URL that can be used to request the result of the completed asynchronous operation. This may or may not be the current request's URL.
```